### ISSUE TITLE
Wont open

### ISSUE THREAD

**Issue by zomile** [KingHabib/CS-Jukebox#12]
Will not open even after installing runtime. Help?

**Comment by KietTezend** [comment]
me too still not open even after installing runtime. Help me please?

**Comment by pinlupe** [comment]
am having the same issue

**Comment by oworope** [comment]
@zzhabib hey can you help us plz

**Comment by zzhabib** [comment]
Make sure the file `gamestate_integration_jukebox.cfg` is in the directory of the program. If that doesn't work, then there must be some other issue. I would attempt to look into this issue more, but unfortunately I don't have much time to maintain this project currently.

**Comment by Speculonse** [comment]
> @zzhabib hey can you help us plz

Did you solve the issue ?

**Comment by oworope** [comment]
> Did you solve the issue ?

No, because my cs2 installation got removed from existence. And what about you, have you tried this solution?

**Comment by TimosCodd** [comment]
I added a logfile for better understanding of the problem. Sorry for the Inactivity on this.
#13

**Comment by lxcoo** [comment]
Has the path to CS2 been correctly configured in properties.json?

**Comment by SameSpecter** [comment]
I have the same problem and i resolve it. How ? It's simple : Maybe your CS Game was intalled in a specific folder or other, so CS Jukebox is unable to find it. For me as exemple, it's on another disc named "D:".

Go look into "**properties.json**" : 
![image](https://github.com/zzhabib/CS-Jukebox/assets/72594972/0ba4e50f-9581-4409-9b13-66d1ed4687e7)
Into that properties, you'll see a path for "C://... something"

**Go into steam > Right Click on CS2 > Local files** ans copy the path to the folder, like that :
![image](https://github.com/zzhabib/CS-Jukebox/assets/72594972/dab60199-db65-4b8f-a122-6f1c1084e40c)
Save, and try, he should be works !

-----
If don't works, look the others .json files, and rectify the path. (Because there is no auto detection of folders)

Have fun !

**Comment by MrDolmin** [comment]
I tried this method, it doesn't work for me BUT I found a simpler solution, If you turn on the application and it does not turn on, just delete this file name "properties.json" and after that open the app and it should work and the jukebox will ask you to indicate the file location of CS2.

PS: I specify that this method works, do not delete again "properties.json" otherwise each time it will ask you to locate your file CS2, and it can get annoying very quickly.

**Comment by innowrun** [comment]
Guys help me please. None of these methods helps( 
I installed the app corectly but after selecting CS2 game folder in does not launching. I tried every solution that you guys wrote here, but nothing helped

**Comment by playertimen** [comment]
im having a fucking same problem it wont open
my version is 1.1

**Comment by slipiei** [comment]
you need to make sure that the path to the game in the "properties.json" is written with "\\\\" not with "\\". It helped me
